cancermuts/datasources.py

```diff
@@ -337,6 +338,9 @@ def add_mutations(self, sequence, metadata=[]):
         if 'entrez' not in sequence.aliases.keys() or sequence.aliases['entrez'] is None:
             self.log.error('Entrez ID alias not available in sequence object')
             raise TypeError('Entrez ID alias not available in sequence object')
+        
+        if hasattr(sequence, "is_canonical") and not sequence.is_canonical:
```

Comment:
are there cases in which an instance of `cancermuts.core.Sequence` doesn't have `is_canonical` set?

in principle there shouldn't be